hawc/apps/assessment/models.py

```diff
@@ -775,11 +776,97 @@ def try_read_df(cls, data, suffix: str, worksheet_name: str = None) -> pd.DataFr
         return df
 
 
+class Job(models.Model):
+    PENDING = 1
```

Comment:
use `IntChoiceEnum` for status and job: https://github.com/shapiromatron/hawc/blob/master/hawc/apps/vocab/models.py#L16